wiki/FAQ/en.md

```diff
@@ -56,161 +77,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
+
+#### Basic explanation
+##### Playfield
+![osu!mania playfield][osu!mania playfield image]
+
+By default, the flow of the hitobjects falls from top to bottom of the convenyor (the starting arrow will show where hitobjects will flow to), with the key control at the bottom and the judgement barline above the key control. To reverse the control and flow of the playfield, it can be changed in the Options sidebar; click on _osu!mania layout_ to set osu!mania keys settings and enable _Vertically flip playfield (DDR style)_ for this key setting manually.
+
+Healthbar is located at the right side of the conveyor belt. Please note that health drain is disabled in osu!mania, so only the hitobjects will affect the healthbar. Combo will not break when playing the keys on empty judgement barline.
+
+##### Tapbars
+![osu!mania's tapbars][osu!mania tapbars image]
+
+Tapbars are the hitcircles of osu!mania. Tapbars are falling bars to be tapped on the judgement barline, with correct key corresponding to each of the tapbar falls into. More correct keys must be tapped simultaneously if the tapbars fall simultaneously.
+
+##### Holdbars
+![osu!mania's Holdbars][osu!mania Holdbars image]
 
-This mode has been widely used in almost all of the major rhythm games that require good hand/leg coordination, where the beat notes with their quantity depending on BPM and difficulty move on the conveyer, and the player have to press the correct key for that specific beat note in time. Though the game mode was originally made as an imitation of Beatmania style of playing, however, by changing the number of keys, you can make it look more like Guitar Hero (5) or Dance Dance Revolution (4) and so on.
+Holdbars are the sliders and spinners of osu!mania. When the holdbar reached the judgement barline, tap the starting tapbar in time with correct key, hold, and release at the ending tapbar of the holdbar.
 
-Currently, you can set from 1 to 9 Key by game modifier (by default, it would be around 4 to 8 Key, except for osu!mania-specific beatmaps) to an upward of 2 - 18 Key (two playfield) with the [Co-Op][Co-Op wikilink] game modifier.
+While holding the holdbar(s), the healthbar will be charged slowly. Depending on the song and mapping quirks, it is possible that other tapbars or holdbars will appear while holding the holdbar(s), so please take that into account.
```

Comment:
I am trying to avoid using the (taken from Wikipedia) second-person personal pronoun, `you`.

Rewrote to: `Healthbar regeneration occurs slowly while holding the holdbar(s).`